<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>KeyUsageWidget</class>
 <widget class="QWidget" name="KeyUsageWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Select which key usage extensions 
the certificate will provide.</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="keyusagegroup">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="title">
      <string>Key &amp;usage extension</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QCheckBox" name="signature_toggle">
        <property name="text">
         <string>Digital Signature</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="key_encipherment">
        <property name="text">
         <string>Key Encipherment</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="key_agreement_toggle">
        <property name="text">
         <string>Key agreement</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="certificate_singing_toggle">
        <property name="text">
         <string>Certificate singing</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="encipher_only_toggle">
        <property name="text">
         <string>Encipher only</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="decipher_only_toggle">
        <property name="text">
         <string>Decipher only</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
